clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -22414,6 +22414,12 @@ RValue CodeGenFunction::EmitIntelFPGAMemBuiltin(const CallExpr *E) {
   return RValue::get(Ann);
 }
 
+static bool hasFuncNameRequestedFPAccuracy(StringRef Name,
+                                           const LangOptions LangOpts) {
```

Comment:
To avoid unnecessary copy
```suggestion
                                           const LangOptions &LangOpts) {
```